Arduino/McLighting/McLighting.ino

```diff
@@ -692,6 +771,105 @@ void setup() {
     getStatusJSON();
   });
 
+  server.on("/pixelconf", []() {
+
+    /*
+
+    // This will be used later when web-interface is ready and HTTP_GET will not be allowed to update the Strip Settings
+
+    if(server.args() == 0 and server.method() != HTTP_POST)
+    {
+      server.sendHeader("Access-Control-Allow-Origin", "*");
+      server.send(200, "text/plain", "Only HTTP POST method is allowed and check the number of arguments!");
+      return;
+    }
+
+    */
+
+    bool updateStrip = false;
+    if(server.hasArg("ct")){
+      uint16_t pixelCt = server.arg("ct").toInt();
+      if (pixelCt > 0) {
+        WS2812FXStripSettings.stripSize = pixelCt;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.printf("/pixels: Count# %d\n", pixelCt);
+      }
+    }
+    if(server.hasArg("rgbo")){
+      String RGBOrder = server.arg("rgbo");
+      DBG_OUTPUT_PORT.print("/pixels: RGB Order# ");
+      if (RGBOrder == "grb") {
+        WS2812FXStripSettings.RGBOrder = NEO_GRB;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "gbr") {
+        WS2812FXStripSettings.RGBOrder = NEO_GBR;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "rgb") {
+        WS2812FXStripSettings.RGBOrder = NEO_RGB;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "rbg") {
+        WS2812FXStripSettings.RGBOrder = NEO_RBG;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "brg") {
+        WS2812FXStripSettings.RGBOrder = NEO_BRG;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else if (RGBOrder == "bgr") {
+        WS2812FXStripSettings.RGBOrder = NEO_BGR;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.println(RGBOrder);
+      } else {
+        DBG_OUTPUT_PORT.println("invalid input!");
+      }
+    }
+    if(server.hasArg("pin")){
+      uint8_t pin = server.arg("pin").toInt();
+      DBG_OUTPUT_PORT.print("/pixels: GPIO used# ");
+      #if defined(USE_WS2812FX_DMA) or defined(USE_WS2812FX_UART1) or defined(USE_WS2812FX_UART2)
```

Comment:
One more thing... Why the outer `if` check? You check `USE_WS2812FX_DMA`, `USE_WS2812FX_UART1` and `USE_WS2812FX_UART2` twice....

Reply:
They are just pre-compiler flags. If one comes one of those methods during compilation, user cannot change the GPIO pin as those methods are hardware limited to those pins on ESP8266. In the future we should be able to switch between these methods, but for now these portions of the code are under pre-processor flags.

Reply:
TODO: Make switching between methods: Adafruit NeoPixel, NeoPixelBus's DMA, NeoPixelBus's UART1 and NeoPixelBus's UART2 as REST API.

Reply:
Added to todo.